test/util/testHttpsConfig.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
What's going on in this config file?

Reply:
I just set everything to a bogus value because I didn't need anything to work. Admittedly it's mostly a waste of space, so I'll just delete everything that I'm not explicitly using.

Reply:
We're pretty rigorous about setting defaults. You could just pass the needed https params to the constructor, that should make it a lot simpler.